a3p-integration/proposals/n:upgrade-next/mintHolder.test.js

```diff
@@ -8,18 +8,18 @@ import {
   getAssetList,
   swap,
   getPSMChildren,
-  upgradeMintHolder,
 } from './test-lib/mintHolder-helpers.js';
 import { networkConfig } from './test-lib/index.js';
 
-test('mintHolder contract is upgraded', async t => {
+test('verify mintHolder contract upgrade', async t => {
   const receiver = await addUser('receiver');
   await provisionSmartWallet(receiver, `20000000ubld`);
 
   let assetList = await getAssetList();
   t.log('List of mintHolder vats being upgraded: ', assetList);
-  await upgradeMintHolder(`upgrade-mintHolder`, assetList);
-  await mintPayment(t, receiver, assetList, 10);
+  // FastUSDC doesn't have a contractKit indexed by its label
```

Comment:
This seems right to me but I'd like @dckc to verify if he's available

Reply:
```suggestion
  // The FastLP mint is not in a mintHolder; it's a ZCFMint in the FastUSDC contract
```